modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -250,6 +250,52 @@ definition {
 		}
 	}
 
+	@description = "LPS-182505 - Verify that it's possible to create an user notification template"
+	@priority = 5
+	test CanCreateUserNotificationTemplate {
+		property portal.acceptance = "true";
+
+		task ("Given the user navigates to Notifications Templates in the Control Panel") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+		}
+
+		task ("When the user creates a new User Notification Template") {
```

Comment:
`When the user adds a new user notification template`

No need to capitalize since it's not a proper noun